<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="camera_tests.TestCamera-20220429231332" tests="7" file="camera_tests.py" time="13.082" timestamp="2022-04-29T23:13:48" failures="0" errors="4" skipped="0">
	<testcase classname="camera_tests.TestCamera" name="test_camera" time="4.935" timestamp="2022-04-29T23:13:40"/>
	<testcase classname="camera_tests.TestCamera" name="test_predict_image" time="1.308" timestamp="2022-04-29T23:13:44"/>
	<testcase classname="camera_tests.TestCamera" name="test_translation" time="1.442" timestamp="2022-04-29T23:13:45"/>
	<testcase classname="camera_tests.TestCamera" name="test_epoch_end" time="1.392" timestamp="2022-04-29T23:13:41">
		<error type="TypeError" message="epoch_end() takes 0 positional arguments but 4 were given"><![CDATA[Traceback (most recent call last):
  File "/usr/lib/python3.9/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.9/unittest/case.py", line 593, in run
    self._callTestMethod(testMethod)
  File "/usr/lib/python3.9/unittest/case.py", line 550, in _callTestMethod
    method()
  File "/home/luka/Desktop/Assignment_6/src/unittest/python/camera_tests.py", line 37, in test_epoch_end
    self.assertEqual(expected, self.model.epoch_end(1, float_num, 0))
TypeError: epoch_end() takes 0 positional arguments but 4 were given
]]></error>
	</testcase>
	<testcase classname="camera_tests.TestCamera" name="test_get_suggestion" time="1.374" timestamp="2022-04-29T23:13:43">
		<error type="AttributeError" message="'TestCamera' object has no attribute 'get_suggestion'"><![CDATA[Traceback (most recent call last):
  File "/usr/lib/python3.9/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.9/unittest/case.py", line 593, in run
    self._callTestMethod(testMethod)
  File "/usr/lib/python3.9/unittest/case.py", line 550, in _callTestMethod
    method()
  File "/home/luka/Desktop/Assignment_6/src/unittest/python/camera_tests.py", line 31, in test_get_suggestion
    self.assertEqual(expected, self.get_suggestion())
AttributeError: 'TestCamera' object has no attribute 'get_suggestion'
]]></error>
	</testcase>
	<testcase classname="camera_tests.TestCamera" name="test_validation_epoch_end" time="1.327" timestamp="2022-04-29T23:13:47">
		<error type="TypeError" message="validation_epoch_end() takes 0 positional arguments but 2 were given"><![CDATA[Traceback (most recent call last):
  File "/usr/lib/python3.9/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.9/unittest/case.py", line 593, in run
    self._callTestMethod(testMethod)
  File "/usr/lib/python3.9/unittest/case.py", line 550, in _callTestMethod
    method()
  File "/home/luka/Desktop/Assignment_6/src/unittest/python/camera_tests.py", line 27, in test_validation_epoch_end
    self.assertEqual(expected, self.model.validation_epoch_end(0))
TypeError: validation_epoch_end() takes 0 positional arguments but 2 were given
]]></error>
	</testcase>
	<testcase classname="camera_tests.TestCamera" name="test_validation_step" time="1.305" timestamp="2022-04-29T23:13:48">
		<error type="TypeError" message="validation_step() takes 0 positional arguments but 1 was given"><![CDATA[Traceback (most recent call last):
  File "/usr/lib/python3.9/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.9/unittest/case.py", line 593, in run
    self._callTestMethod(testMethod)
  File "/usr/lib/python3.9/unittest/case.py", line 550, in _callTestMethod
    method()
  File "/home/luka/Desktop/Assignment_6/src/unittest/python/camera_tests.py", line 23, in test_validation_step
    self.assertEqual(expected, self.model.validation_step(), "Testing Test_validation_step()")
TypeError: validation_step() takes 0 positional arguments but 1 was given
]]></error>
	</testcase>
</testsuite>
